GroupIS1024/Pages/Teams/Index.cshtml

```diff
@@ -37,22 +37,40 @@
         </div>
     </div>
 
-    <table class="table">
+    <table class="table border-primary table-hover">
         <thead>
             <tr>
+                <th>
+                    @Html.DisplayNameFor(model => model.Team[0].TeamId)
+                </th>
                 <th>
                     @Html.DisplayNameFor(model => model.Team[0].TeamName)
                 </th>
+                <th>
+                    @Html.DisplayNameFor(model => model.Team[0].Coaches)
+                </th>
+                <th>
+                    @Html.DisplayNameFor(model => model.Team[0].Players)
+                </th>
```

Comment:
How does this decrease technical debt and/or increase alignment with best practices?